core/templates/dev/head/pages/exploration-editor-page/feedback-tab/services/thread-data.service.ts

```diff
@@ -182,6 +182,7 @@ angular.module('oppia').factory('ThreadDataService', [
           thread.status =
             action === ACTION_ACCEPT_SUGGESTION ? STATUS_FIXED : STATUS_IGNORED;
           _openThreadsCount -= 1;
+          _fetchMessages(threadId);
```

Comment:
Since this is blocking release, you can just return this value.

However, please file an issue and leave a TODO so we don't forget to change this to use the cache.

Reply:
@brianrodri Ok